Report.md

```diff
@@ -164,27 +160,21 @@ These diagrams represents the dynamic behaviour of objects and the structure of
 Potential Advantages and Disadvantages
 ======================================
 ## **Competitive Advantage**
-Developing this new information system, if successful, will lead to a plethora of competitive advantages against other major travel companies looking to increase their customer loyalty or retention. Currently, the major travel company involved is in a great position in the market but has some areas that it is lacking in. For example, the CMC (Call Management Centre) has reduced operating efficiency due to call flow control and call routing issues. This leads to some customers having long wait times and drives up the inbound call cost of the CMC. Through the information system, these areas could be improved and provide many advantages.
```

Comment:
"Currently, the major travel company involved " for this maybe reword it to help the flow "Current the major travel company is in a position to invest resources into areas they are lacking in."

Reply:
I cut that whole sentence for word count, are you suggesting we bring it back in?

Reply:
Oops I didn't realise it was cut out you can get rid of it